### PR TITLE
Docs : fixed grammer

### PR DESCRIPTION
**Pull request by programmersoham**
### Description

> Describe your PR! If it fixes specific issue, mention it with "Fixes # (issue)".

### Select what type of change this PR introduces:

1. [x] **Just code/docs improvement** (no functional change). 
2. [ ] **Bug fix** (non-breaking change which fixes an issue).
3. [ ] **New feature** (non-breaking change which adds functionality).
4. [ ] **Breaking change** (fix or feature that would cause existing functionality to not work as expected).

### Update Waspc ChangeLog and version if needed
If you did a bug fix, new feature, or breaking change, that affects waspc, make sure you satisfy the following:
1. [ ] I updated [ChangeLog.md](https://github.com/wasp-lang/wasp/blob/main/waspc/ChangeLog.md) with description of the change this PR introduces.
2. [ ] I bumped waspc version in [waspc.cabal](https://github.com/wasp-lang/wasp/blob/main/waspc/waspc.cabal) to reflect changes I introduced, with regards to the version of the latest wasp release, if the bump was needed.

### CODE REVIEW

**Comment by programmersoham** [comment]
@matijaSos take a look 👀

**Comment by infomiho** [comment]
@programmersoham hey, I would just remove the added `.history` folder, probably something local to your IDE :)

**Comment by programmersoham** [comment]
Sure thanks, I'll change it

**Comment by programmersoham** [comment]
@infomiho Kindly take a look again

**Comment by infomiho** [comment]
@programmersoham could you please sort out the conflicts?

File: web/docs/tutorials/todo-app/06-auth.md [hunk 1]
```diff
@@ -205,11 +205,11 @@ psl=}
 // ...
 ```
 
-We modified entities by adding the User-Task relation, so let's run
+We modified entities by adding the User-Task relation. To create a database schema migration and apply it to the database, let's run the following command  :
```
Comment (infomiho) [inline web/docs/tutorials/todo-app/06-auth.md:208]: We don't usually put space before the `:`, so I would suggest to remove it.

```suggestion
We modified entities by adding the User-Task relation. To create a database schema migration and apply it to the database, let's run the following command:
```

**Comment by programmersoham** [comment]
@infomiho Please help me resolve the conflict

**Pull request closed without merging**